### DISCUSSION TITLE
macOS Sequoia support is ready, but...

### DISCUSSION THREAD

**Discussion by VisualEhrmanntraut** [ChefKissInc/NootedRed#275] [Announcements]
I'm still waiting for Acidanthera to add the Sequoia constants to Lilu. So I can't actually add them in properly. For the meantime, have some screenshots.

![Screenshot 2024-06-12 at 21 19 32](https://github.com/ChefKissInc/NootedRed/assets/30368284/b79e028c-9950-40a0-a042-b26a1ec9d2d4)

![2024-06-13 03 05 06](https://github.com/ChefKissInc/NootedRed/assets/30368284/2817daa2-6cf4-4b8e-a2f0-d0392be0d5de)

**Comment by SalmonPete** [comment]
Great work @VisualEhrmanntraut, I have installed Sequoia on my 4700G system but can't boot to desktop with NootedRed.kext enabled. Any chance of getting an invite to the Telegram group?.

![Screenshot 2024-06-13 at 10 05 04 PM](https://github.com/ChefKissInc/NootedRed/assets/143473778/e7b320a8-2500-422a-bc29-46b216969207)
![Screenshot 2024-06-13 at 10 05 38 PM](https://github.com/ChefKissInc/NootedRed/assets/143473778/d61675fd-4a14-416f-acd3-b3806b6c509f)

**Reply by VisualEhrmanntraut** [reply]
The screenshots are from a secret testing group, which only consists of trusted people. This feature won't be released until Acidanthera actually updates Lilu. Though at this point I'm starting to think that they're dead or something.

**Reply by SalmonPete** [reply]
Glad to see EXtremeXT, or his laptop at least, is still part of the not so secret team. Respect!.

**Reply by VisualEhrmanntraut** [reply]
It's a recent thing, with the new ForgedInvariant and other improvements he decided to try out Hackintosh again.

**Reply by SalmonPete** [reply]
A few people in the Hackintosh community will be very happy at this development. I included the ForgedInvariant.kext in my Sequoia EFI, really plug and play?, no more forgetting if it's 15 or 16 for 8 core Ryzen?.

**Reply by VisualEhrmanntraut** [reply]
Yes ForgedInvariant is fully plug and play and works way better than all the other TSC plug-ins combined.

**Comment by Shaneee** [comment]
Lilu updated for Sequoia if you want it.

`static constexpr KernelVersion maxKernel {KernelVersion::Sequoia};`

`	Sonoma        = 23,
	Sequoia		  = 24,`

[Lilu-1.6.8-DEBUG.zip](https://github.com/user-attachments/files/15841823/Lilu-1.6.8-DEBUG.zip)

**Comment by AlfCraft07** [comment]
I also have a repo at AlfCraft07/LiluSequoia, if you want you can hook the workflow to it or another repo instead of Acidanthera's and build the kext. Or you can just upload the debug version of the updated Lilu and base on that to build, you can download it even from the actions in Acidanthera's repo

**Comment by VisualEhrmanntraut** [comment]
macOS Sequoia support has been added to NootedRed and NootRX as the constants have been added to Lilu a few moments ago.

**Reply by elfametesar** [reply]
I have tried out both the nightly release and the latest one in actions on 15.0, it doesn't load the display with the nightly and it doesn't activate graphics accelaration with the latest one in the actions. Am I supposed to do something else or use something different?

**Reply by VisualEhrmanntraut** [reply]
Add `-lilubetaall` to boot args

**Reply by elfametesar** [reply]
Yeah, it just reboots right when it's about to load the display with -lilubetaall. I'll keep trying

**Reply by VisualEhrmanntraut** [reply]
what kexts are you using

**Reply by elfametesar** [reply]
![image](https://github.com/user-attachments/assets/54f6b5ba-2c52-438b-8752-de06170d2d42)
![image](https://github.com/user-attachments/assets/648e4bcf-d6e5-433c-acf5-9bb27839f71c)

nootedredmine is from actions and the other is the nightly. tried with both
 
boot args: -v debug=0x100 -lilubetaall nv_disable=1 keepsyms=1 alcid=66 -vi2c-force-polling watchdog=0

**Reply by elfametesar** [reply]
I have disabled some of the Lilu plugins that weren't crucial, I booted it up fine now. Thanks
